alternator/executor.cc

```diff
@@ -1358,11 +1358,27 @@ mutation put_or_delete_item::build(schema_ptr schema, api::timestamp_type ts) co
 // s_default_timeout is overwritten in alternator::controller::start_server()
 // based on the "alternator_timeout_in_ms" configuration parameter.
 db::timeout_clock::duration executor::s_default_timeout = 10s;
+db::timeout_clock::duration executor::default_getitem_timeout = 3s;
+db::timeout_clock::duration executor::default_putitem_nonlwt_timeout = 2s;
+db::timeout_clock::duration executor::default_putitem_lwt_timeout = 5s;
+db::timeout_clock::duration executor::default_query_timeout = 10s;
 void executor::set_default_timeout(db::timeout_clock::duration timeout) {
     s_default_timeout = timeout;
 }
-db::timeout_clock::time_point executor::default_timeout() {
-    return db::timeout_clock::now() + s_default_timeout;
```

Comment:
Can you please rebase your patch to the latest master, but more importantly, base your new timeout variables and functions on the new code, not ancient code?
More than a year ago, in commit 69c21f490aa4c6b58fa7f174b70158cd782ab30b, @tchaikov made the timeouts "live updatable", meaning that the timeouts can be changed dynamically while Scylla is running and not just set during boot. This means that the s_default_timeout (now called s_default_timeout_in_s) points to a configuration variable, that can continue changing after first set. Whatever new variant timeouts you invent, should follow the same principles.